src/components/DoNotSellExplainer.tsx

```diff
@@ -32,12 +32,36 @@ export function DoNotSellExplainer({
     event.preventDefault();
     airgap.setConsent(event, { SaleOfInfo: checked });
     setConsentLocal(checked);
+    setSaving(true);
+    setTimeout(() => {
+      setSaving(false);
+      setTimeout(() => {
+        setSaving(undefined);
+      }, 2000);
+    }, 1000);
```

Comment:
Since this is saved instantaneously, why introduce the artificial "Saving..." state for 1 second? I understand why we'd render a confirmation message "Successfully saved" for 2 seconds, but don't see the case for "Saving..."

Reply:
I kept "Saving" -> "Saved" but dropped the part where the confirmation is hidden afterward, which is an accessibility issue.